modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -84,6 +84,28 @@ $ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaceroutings.
 $ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaces.workspace.devfile.io
 ----
 +
+[source,terminal]
+----
+$ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspacetemplates.workspace.devfile.io
+----
++
+[source,terminal]
+----
+$ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaceoperatorconfigs.controller.devfile.io
+----
++
+[WARNING]
+====
+The DevWorkspace Operator provides custom resource definitions (CRDs) that use conversion webhooks. Failing to remove the CRDs above can cause issues on the cluster. The following steps will remove the server that handles these conversion webhooks.
+====
```

Comment:
Two suggestions for this note:
1. "The DevWorkspace Operator provides custom resource definitions (CRDs) that use conversion webhooks. Failing to remove the CRDs can cause issues on the cluster." This part of the note should go as a warning right under the step "Remove the CRDs used by the Operator:"
Pls. note that I have removed the word 'above' from the warning and made the Operator title case in the step.
2. This step should be modified from: "Remove the `DevWorkspace-Webhook-Server` deployment, mutating, and validating webhooks:" to "Remove the `DevWorkspace-Webhook-Server` deployment, mutating, and validating webhooks that handle the conversion webhooks." That will take care of the last sentence in the warning.

Reply:
These are OpenShift object names, which are case-sensitive. I think it may be less clear to use proper title case here when the object on the cluster is named `devworkspace-webhook-server`.

Reply:
I've moved the warning up as suggested and dropped the piece about conversion webhooks, as this is technically no longer accurate. It's the main controller deployment that services these webhooks, and it is removed when the operator is uninstalled.

Reply:
@amisevsk I agree. We need to change the case of the word 'operator' to 'Operator' when it is part of the general text as per OCP convention. If it is part of the name of a CRD, or part of a command, we need to retain it as is.

Reply:
:+1: thanks, I misunderstood the intention on `operator` vs `Operator` -- should be fixed now.